apps/heft/src/plugins/ActionConfigurationLoaders/ActionConfigurationFilesPluginBase.ts

```diff
@@ -174,7 +174,7 @@ export abstract class ActionConfigurationFilesPluginBase implements IHeftPlugin
 
   private async _updateWebpackConfigurationAsync(
     heftConfiguration: HeftConfiguration,
-    bundleProperties: IBundleStageProperties
+    bundleProperties: IBundleSubstageProperties
```

Comment:
What is the difference between a "stage" and a "substage"?

Reply:
We'd previously called things like "pre-compile," "compile," "bundle," and "post-bundle" build "stages." Now that we're calling the larger operations "stages," I've called the things that make up the steps build runs "substages"